### PR TITLE
Own Faction Filter when on Jail Page

### LINKED ISSUE #313: [BUG] My own faction filter on Jail page does not work.
**Description**
When on the Jail page, the Torn Tools UI defaults to my faction "Equilibrium" in the Faction filter, but this filter value does not work. It does not show my faction members when they are in jail. The other factions in the list work properly. 

I believe the issue is because the value of the option for my faction is the full name equilibrium instead of its tag, as is the case with all of the other factions in the generated list based on who is in jail at the time. See the inspector panel in the attached screenshot.

**To Reproduce**
Go to Jail page when someone from your faction is in Jail. Note that the list is empty even though your own faction is preselected in the Faction filter.

**Expected behavior**
I expect to see my faction members listed.

**Screenshots**
Screen Shot 2021-01-06 at 11 58 02 AM

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.7
 - Browser: Both Firefox 84.0.1 and Chrome 87.0.4280.88

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #313 .

### CODE REVIEW

File: torntools/scripts/content/jail/ttJail.js [hunk 1]
```diff
@@ -66,6 +66,8 @@ function addFilterToTable(list, title) {
 
 		filter_container.find("#tt-faction-filter").appendChild(option);
 	}
+	let ownFacOption = doc.new({ type: "option", value: userdata.faction.faction_tag, text: userdata.faction.faction_tag});
```
Comment (DeKleineKobini) [inline torntools/scripts/content/jail/ttJail.js:69]: Own faction should be above the manual factions imo

Comment (Sashank999) [reply torntools/scripts/content/jail/ttJail.js]: Done.

Comment (DeKleineKobini) [reply torntools/scripts/content/jail/ttJail.js]: You misunderstood, you put it even lower instead of higher.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
Check my response on the message.

File: torntools/scripts/content/jail/ttJail.js (hunk 1 above)
Comment (Sashank999) [reply torntools/scripts/content/jail/ttJail.js]: At the last of the list of factions gathered ?

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**